index.html

```diff
@@ -1,13 +1,233 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="h-100" lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>The Social Network</title>
     <link rel="stylesheet" href="/dist/css/style.css" />
   </head>
-  <body>
-    <h1 class="test">test</h1>
+  <body class="d-flex flex-column h-100">
```

Comment:
```suggestion
  <body class="d-flex flex-column mh-vh-100">
```

This is a better approach generally speaking. It might not work for you.